### DISCUSSION TITLE
IDEA!

### DISCUSSION THREAD

**Discussion by s0urce-c0de** [3kh0/ext-remover#1104] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Add a sh1mmer payload that mounts the root partition and modifies crosh to open a menu to choose `crosh` or `bash` instead of just opening `crosh`. Or maybe replace a binary like `top`? Can someone make a sh1mmer payload for me (i don't know how to make payloads or which partition is root/how to mount it). 
**EDIT:**
Mabye also add a SUID binary to enter root. I have a rudimentary one [here](https://github.com/s0urce-c0de/sudo-wrapper) (sorry, I don't code in C. this was my best try)

**Comment by akabutnicer** [comment]
I mean that's a pretty simple feature. I'm not that experienced in bash neither do I have proper experience with sh1mmer, but I'm quite sure you can do that very easily, in fact you could just host your own version of shimmer with that option.

**Reply by s0urce-c0de** [reply]
But I can't figure out which partition is root, or how to mount it (bad superblock or something on nearly all partitions)

**Reply by akabutnicer** [reply]
Fair. I wouldn't know either, perhaps @AshtonDavies could help you with this one.

**Reply by AshtonDavies** [reply]
> But I can't figure out which partition is root, or how to mount it (bad superblock or something on nearly all partitions)

![image](https://github.com/3kh0/ext-remover/assets/117958718/ff72ee32-f9e0-4c4c-b38c-936d1a61ab85)

Number 3 is rootA. Does that answer that part of your question?

**Reply by s0urce-c0de** [reply]
@AshtonDavies how can I mount this? Also maybe we can use a modified recovery image? We can [enable_rw_mount](https://source.chromium.org/chromiumos/chromiumos/codesearch/+/main:src/platform/vboot_reference/scripts/image_signing/common_minimal.sh;l=367-385), modify it, [disable_rw_mount](https://source.chromium.org/chromiumos/chromiumos/codesearch/+/main:src/platform/vboot_reference/scripts/image_signing/common_minimal.sh;l=403-421). I can't do bash good.

**Reply by akabutnicer** [reply]
I mean, I'd at least fully learn bash before attempting to modify something as advanced as sh1mmer.

**Reply by s0urce-c0de** [reply]
@akabutnicer I am good with python3

**Reply by AshtonDavies** [reply]
> I am good with python3

Works I guess. Probably still better done with bash.

**Reply by s0urce-c0de** [reply]
@AshtonDavies can you write something up and test (my school took chromebooks away) best bet is probably modify recovery image.

**Reply by AshtonDavies** [reply]
What exactly do you need to mount root SUID and access bash for?

**Reply by s0urce-c0de** [reply]
root [SUID](https://en.wikipedia.org/wiki/Setuid) is a binary that runs as root. access bash so I can install chromebrew and install packages like minecraft (latest).

**Reply by AshtonDavies** [reply]
> root [SUID](https://en.wikipedia.org/wiki/Setuid) is a binary that runs as root. access bash so I can install chromebrew and install packages like minecraft (latest).

Yeah I was just curious because RMA hacks are patched, but that's pretty cool I guess. Chromebooks generally take a long time to launch Minecraft Java Edition, and it runs pretty bad even with optimizations.

**Reply by s0urce-c0de** [reply]
Maybe we can install firefox or something?

**Reply by AshtonDavies** [reply]
> Maybe we can install firefox or something?

Or just run Firefox on a website with Python libraries.

**Reply by s0urce-c0de** [reply]
My point is we have a free linux machine with root. And we can install unblocked browsers **without** any restrictions/being blocked.
